source/2019-04-05-the-ember-times-issue-93.md

```diff
@@ -14,18 +14,43 @@ responsive: true
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
-<change section title emoji>
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+## Reason #1. [Community](https://emberjs.com/community) 🏆
+
+One great reason to use Ember is the [community](https://emberjs.com/community) that you are/will be a part of. Ember provides a wide platform for everyone to be an active learner and contributor. Where will you discover yourself next?
+
+This year, we surveyed 10 different ways to [learn](https://emberjs.com/ember-community-survey-2019/#MS_Q103) and [contribute](https://emberjs.com/ember-community-survey-2019/#MS_Q112)!
+
+Our 5 most favorite places to learn: (drumrolls, please)
+
+- [Documentations](https://guides.emberjs.com/release/) (71%)
+- [Ember.js blog](https://blog.emberjs.com/) (54%)
+- [RFCs](https://github.com/emberjs/rfcs) (52%)
+- Code samples (51%)
+- Blog posts (47%)
+
+And our 5 most favorite ways to give back: 🥁🥁🥁
+
+- Attend Meetups and conferences (35%)
+- Open issues (35%)
+- Maintain addons (27%)
+- Open PRs (24%)
+- Answer community questions (16%)
+
+I think it's great to see that we like to meet in person and share knowledge. Each year, you can meet Ember devs around the world at [EmberConf](https://emberconf.com/), [EmberCamp](http://embercamp.com/), and [EmberFest](https://emberfest.eu/). You can also meet them on a regular basis through [Meetups](https://emberjs.com/community/meetups/).
```

Comment:
```suggestion
It's great to see that we all like to meet in person and share knowledge with each other. Each year, you can meet Ember devs from around the world at [EmberConf](https://emberconf.com/), [EmberCamp](http://embercamp.com/), and [EmberFest](https://emberfest.eu/). You can also meet them on a regular basis through [Meetups](https://emberjs.com/community/meetups/).
```

Changed the "I" to more of a "we" :) since it flows better with the rest of the content

Reply:
Thanks!